### ISSUE TITLE
response Still getting this same error on 1.3.2

### ISSUE THREAD

**Issue by anotherso1a** [hilongjw/vue-lazyload#395]
Still getting this same error on 1.3.2

_Originally posted by @palfaro91 in https://github.com/hilongjw/vue-lazyload/pull/381#issuecomment-523226397_

I use this code fixed it

```js
Vue.prototype.destroy = Vue.prototype.$destroy
```

**Comment by hansenbeautyspotter** [comment]
@palfaro91 you put it where you usually put any prototype or your main instance of Vue is being defined, if your Vue is created with CLI or the Templates usually the main instance is in main.js

**Comment by palfaro91** [comment]
If I'm working with Nuxt would I place this code in the plugin file? @hansenbeautyspotter

**Comment by hansenbeautyspotter** [comment]
@palfaro91 yeah I think you can just add it in the plugin file since the idea of the fix is to give Vue new prototype / wrap the destroy function for the VueLazyLoad.

`
import Vue from 'vue'
import VueLazyload from 'vue-lazyload'

Vue.prototype.destroy = Vue.prototype.$destroy // add here

Vue.use(VueLazyload)
`
